pype/plugins/global/publish/integrate_new.py

```diff
@@ -929,16 +929,25 @@ def prepare_file_info(self, path, size=None, file_hash=None, sites=None):
         Returns:
             rec: dictionary with filled info
         """
+        local_site = 'studio'  # default
+        remote_site = None
+        sync_server_presets = None
         try:
-            sync_server_presets = config.get_presets()["sync_server"]["config"]
+            settings = pype.api.get_current_project_settings()
+            sync_settings = settings.get("global")["sync_server"]
```

Comment:
I think this can be `settings["global"]["sync_server"]`

Reply:
I agree, probably better if it errors out on us, if the settings are missing.